data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -430,7 +432,7 @@
                                                                                     [/column]
                                                                                     [column]
                                                                                         [label]
-                                                                                            label="↔"
+                                                                                            label= _ "↔"
```

Comment:
Does anyone know what this is used for? I think if it's going to be translatable then it needs a po comment and/or a context prefix.

Reply:
Turn off markcheck instead.